README.md

```diff
@@ -11,36 +11,16 @@ Allows for importing [MyAir](https://myair.resmed.com/) data to [InfluxDB](https
 
 ## Setup
 
-Choose one of these 3 methods.
```

Comment:
let's keep the ability to have separate influx and grafana separately (meaning: add a new method instead of replacing existing methods)